source/routing/tests/routing_api_test.cpp

```diff
@@ -799,10 +799,11 @@ BOOST_FIXTURE_TEST_CASE(direct_path_under_the_limit, direct_path_routing_api_dat
 
     auto sp = worker.find_nearest_stop_points(a_s_dur - navitia::seconds(1), b.data->pt_data->stop_point_proximity_list, false);
 
-    BOOST_REQUIRE_EQUAL(sp.size(), 1);
-    auto res_sp = sp[0];
-    BOOST_CHECK_EQUAL(res_sp.second, to_duration(B.distance_to(S), navitia::type::Mode_e::Walking));
-    BOOST_CHECK_EQUAL(b.data->pt_data->stop_points[res_sp.first]->name, "stop_point:stopB");
+    navitia::routing::map_stop_point_duration tested_map;
+    tested_map.insert(navitia::routing::stop_point_duration(
+                        navitia::routing::SpIdx(*(b.data->pt_data->stop_points_map["stop_point:stopB"])),
+                        to_duration(B.distance_to(S), navitia::type::Mode_e::Walking)));
+    BOOST_CHECK_EQUAL_COLLECTIONS(sp.begin(), sp.end(), tested_map.begin(), tested_map.end());
```

Comment:
`sp.cbegin()/.cend()` ..... 

http://stackoverflow.com/questions/12001410/what-is-the-reason-behind-cbegin-cend